modules/js-and-css/embed-optimizer/hooks.php

```diff
@@ -21,7 +21,7 @@
  * @param string $html The oEmbed HTML.
  * @return string
  */
-function perflab_optimize_oembed_html( string $html ): string {
+function embed_optimizer_html( string $html ): string {
```

Comment:
Just a naming nit: shouldn't this be something like `embed_optimizer_filter_oembed_html`?

Reply:
missed that one, fixing...